src/atoms/buttonLink.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+const LinkAsAButton = ({
+  href,
+  alt,
+  rel,
+  target,
+  disabled,
+  children,
+  className,
+  size,
+  referrerPolicy,
+  onClick = () => {},
+}) => (
+  <a
+    rel={rel}
+    target={target}
+    onClick={onClick}
+    referrerPolicy={referrerPolicy}
+    className={className}
+    href={href}
+  >
+    {children}
+  </a>
+);
+
+const ClassicButton = ({
+  type,
+  onClick,
+  alt,
+  size,
+  disabled,
+  referrerPolicy,
+  className,
+  children,
+}) => (
+  <button onClick={onClick} type={type} className={className}>
+    {children}
+  </button>
+);
+
+const ButtonLink = ({
+  href,
+  alt,
+  rel,
+  target,
+  disabled,
+  children,
+  size,
+  type,
+  onClick,
+  referrerPolicy,
+  className,
+}) => {
+  if (href) {
+    return (
+      <LinkAsAButton
+        href={href}
+        rel={rel}
+        target={target}
+        alt={alt}
+        disabled={disabled}
+        children={children}
+        size={size}
+        onClick={onClick}
+        className={className}
+        referrerPolicy={referrerPolicy}
+      />
+    );
+  }
```

Comment:
you could make a great use of spread operator like so : 

```suggestion
const ButtonLink = ({
  href,
  type,
  onClick,
  ...props,
}) => {
  if (href) {
    return (
      <LinkAsAButton
        href={href}
        {...props}
      />
    );
  }
```

this way, future addition of props should be easier

Reply:
plus opaque en l'absence de typescript mais ok